src/test/java/PetTest.java

```diff
@@ -18,4 +18,19 @@ public void addPetReturnName(){
 
```

Comment:
Testen zien er netjes uit! 
Volgens mij is de test van regel 11 een getNamereturnsName test en niet een add Pet.
Misschien is het nog leuk om een aantal extra testen te doen? Om bijvoorbeeld ook de owner te testen? Zoals getOwnerReturnsNameOwner en een add/set Owner? :)